lan-mouse-ipc/src/lib.rs

```diff
@@ -219,7 +222,7 @@ pub enum FrontendEvent {
     ConnectionAttempt { fingerprint: String },
 }
 
-#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
+#[derive(Debug, FloatEq, PartialEq, Clone, Serialize, Deserialize)]
```

Comment:
We don't need Eq here, so no need to pull in this crate. In fact I don't even think we really need PartialEq, its more of a just in case it might be useful for some frontend.